### PR TITLE
LPS-133422 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-133422
cc @Tim-Cao

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase [hunk 1]
```diff
@@ -32,6 +32,73 @@ definition {
...
+			Page.gotoFriendlyURLHistory();
+
+			Page.viewActiveURL(activeURL = "/test-content-page-name");
+		}
+
+		task ("Delete the content page in the default site") {
+			JSONLayout.deletePublicLayout(
+				groupName = "Guest",
+				layoutName = "Test Content Page Name");
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase:99]: Unneeded

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase [hunk 2]
```diff
@@ -32,6 +32,73 @@ definition {
...
+		task ("Assert the friendly URL") {
+			SitePages.openPagesAdmin(siteURLKey = "test-site-name");
+
+			SitePages.gotoPageMenuItem(
+				menuItem = "Configure",
+				pageName = "Test Content Page Name");
+
+			Page.gotoFriendlyURLHistory();
+
+			Page.viewActiveURL(activeURL = "/test-content-page-name");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase:64]: View friendly URL on content page at view mode

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase [hunk 3]
```diff
@@ -32,6 +32,73 @@ definition {
...
+		task ("Assert the friendly URL into Default Site") {
+			SitePages.openPagesAdmin(siteURLKey = "guest");
+
+			SitePages.gotoPageMenuItem(
+				menuItem = "Configure",
+				pageName = "Test Content Page Name");
+
+			Page.gotoFriendlyURLHistory();
+
+			Page.viewActiveURL(activeURL = "/test-content-page-name");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase:92]: View friendly URL on content page at view mode

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase [hunk 4]
```diff
@@ -32,6 +32,73 @@ definition {
...
+	@description = "This is a test for LPS-128151. Export and import a site containing a content page into the default site."
+	@priority = "4"
+	test ExportImportSiteIntoDefaultSite {
+		property portal.acceptance = "true";
+
+		task ("Add a Content Page via JSON") {
+			JSONLayout.addPublicLayout(
+				groupName = "Test Site Name",
+				layoutName = "Test Content Page Name",
+				type = "content");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase:44]: Add a content page named Home

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase [hunk 5]
```diff
@@ -32,6 +32,73 @@ definition {
...
+
+			Page.gotoFriendlyURLHistory();
+
+			Page.viewActiveURL(activeURL = "/test-content-page-name");
+		}
+
+		task ("Export the created test site") {
+			LAR.exportSiteCP(siteScopeName = "Test Site Name");
+
+			var key_larFileName = "${larFileName}";
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase:70]: Redundant step

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase [hunk 6]
```diff
@@ -32,6 +32,73 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-128151. Export and import a site containing a content page into the default site."
+	@priority = "4"
+	test ExportImportSiteIntoDefaultSite {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase:37]: ImportSiteWithHomePageToDefaultSite

**Comment by Tim-Cao** [comment]
Hi @ozysouza, Could you update the case according to comments?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase (hunk 1 above)
Comment (ozysouza) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase:99]: Hi @Tim-Cao, just to double confirm, if I remove this step it'll let the content page in the default site, is it ok? the teardown isn't excluding it.

**Comment by ozysouza** [comment]
> Hi @ozysouza, Could you update the case according to comments?

Hi @Tim-Cao PR updated.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase [hunk 7]
```diff
@@ -469,4 +469,55 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-128151. Export and import a site containing a home content page into the default site."
+	@priority = "4"
+	test ImportSiteWithHomePageToDefaultSite {
+		property portal.acceptance = "true";
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase:475]: Unneeded property

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase [hunk 8]
```diff
@@ -469,4 +469,55 @@ definition {
...
+		}
+
+		task ("View friendly URL on content page at view mode") {
+			ContentPagesNavigator.openViewContentPage(
+				pageName = "Home",
+				siteName = "Test Site Name");
+
+			var portalURL = PropsUtil.get("portal.url");
+
+			AssertLocation(value1 = "${portalURL}/web/test-site-name/home");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase:497]: AssertLocation.assertPartialLocation(value1 = "/home");

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase [hunk 9]
```diff
@@ -469,4 +469,55 @@ definition {
...
+				larFileName = "${larFileName}",
+				siteName = "guest");
+		}
+
+		task ("View friendly URL on content page at view mode") {
+			ContentPagesNavigator.openViewContentPage(
+				pageName = "Home",
+				siteName = "Guest");
+
+			AssertLocation(value1 = "${portalURL}/web/guest/home");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase:519]: AssertLocation.assertPartialLocation(value1 = "/home");

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase [hunk 10]
```diff
@@ -469,4 +469,55 @@ definition {
...
+	test ImportSiteWithHomePageToDefaultSite {
+		property portal.acceptance = "true";
+
+		task ("Add a Content Page via JSON") {
+			JSONLayout.addPublicLayout(
+				groupName = "Test Site Name",
+				layoutName = "Home",
+				type = "content");
+		}
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase:483]: Add some content to page like a fragment

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase [hunk 11]
```diff
@@ -469,4 +469,55 @@ definition {
...
+		task ("View friendly URL on content page at view mode") {
+			ContentPagesNavigator.openViewContentPage(
+				pageName = "Home",
+				siteName = "Test Site Name");
+
+			var portalURL = PropsUtil.get("portal.url");
+
+			AssertLocation(value1 = "${portalURL}/web/test-site-name/home");
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase:498]: Add step to view fragment on page

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase [hunk 12]
```diff
@@ -469,4 +469,55 @@ definition {
...
+
+		task ("View friendly URL on content page at view mode") {
+			ContentPagesNavigator.openViewContentPage(
+				pageName = "Home",
+				siteName = "Guest");
+
+			AssertLocation(value1 = "${portalURL}/web/guest/home");
+		}
+	}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase:521]: Add step to view fragment on page

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase [hunk 13]
```diff
@@ -469,4 +469,55 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-128151. Export and import a site containing a home content page into the default site."
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase:472]: The user can import a site with Home page into the default site

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase [hunk 14]
```diff
@@ -469,4 +469,55 @@ definition {
...
+			JSONLayout.addPublicLayout(
+				groupName = "Test Site Name",
+				layoutName = "Home",
+				type = "content");
+		}
+
+		task ("Publish the content page") {
+			JSONLayout.publishLayout(
+				groupName = "Test Site Name",
+				layoutName = "Home");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase:487]: Use UI step.  The json one is used for when no need to add content to page.

**Comment by Tim-Cao** [comment]
Hi @ozysouza, Could you update again? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, Could you update again? Thanks

Hi @Tim-Cao, PR updated.

**Comment by Tim-Cao** [comment]
See changes https://github.com/liferay-echo/liferay-portal/pull/5106/commits/ebd5a36fc1095e8089c641a8432a55bca4b65bcf

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**